vital/types/object_track_set.h

```diff
@@ -113,15 +113,14 @@ class VITAL_EXPORT object_track_state : public track_state
     return time_;
   }
   
-  ///Vector getters/setters for missing kw18 fields
+  ///Vector getters/setters for kw18 fields
```

Comment:
I wouldn't call these kw18 fields either, vital types should be abstracted away from file formats, algorithms, etc...